AFQ/dki.py

```diff
@@ -8,22 +8,48 @@
 from dipy.core import gradients as dpg
 
 
-def fit_dki(data_files, bval_files, bvec_files, mask=None, min_d=0, max_d=3,
-            out_dir=None):
+def fit_dki(data_files, bval_files, bvec_files, mask=None, min_kurtosis=-1,
+            max_kurtosis=3, out_dir=None):
     """
-    Fit the DKI model, save files with parameters and derived maps
+    Fit the DKI model, save files with derived maps
 
     Parameters
     ----------
     data_files : str or list
-        if str, that's the full path
-        if list, each entry is a full path
+        Files containing DWI data. If this is a str, that's the full path to a
+        single file. If it's a list, each entry is a full path.
     bval_files : str or list
-        ditto
+        Equivalent to `data_files`.
     bvec_files : str or list
-        ditto
+        Equivalent to `data_files`.
+    mask : ndarray, optional
+        Binary mask, set to True or 1 in voxels to be processed.
+        Default: Process all voxels.
+    min_kurtosis : float, optional
+        The minimal plausible value of kurtosis. Default: -1.
+    max_kurtosis : float, optional
+        The maximal plausible value of kurtosis. Default: 3.
```

Comment:
I tend to leave out default values; I believe these get published in the auto-generated function declaration comment, and this can get stale.